packages/transducers/test/flatten.ts

```diff
@@ -14,6 +14,11 @@ describe("flatten", () => {
     it("strings", () => {
         assert.deepEqual([...flatten(["", "a"])], ["", "a"]);
         assert.deepEqual([...flatten([[], ["a"], ""])], ["a", ""]);
+        assert.deepEqual([...flatten([["abc"]])], ["abc"]);
+        assert.deepEqual([...flatten(["abc"])], ["abc"]);
+        assert.deepEqual([...flatten("abc")], ["abc"]);
+        assert.deepEqual([...flatten([""])], [""]);
+        assert.deepEqual([...flatten("")], [""]);
```

Comment:
This should be in a separate group saying "treats strings as atomic".  (I thought I had done that...)

Reply:
That's fine - will move those myself! Thank you!